src/SSMProblems.jl

```diff
@@ -1,40 +1,203 @@
 """
-A unified interface to define State Space Models interfaces in the context of Particle MCMC algorithms.
+A unified interface to define state space models in the context of particle MCMC algorithms.
 """
 module SSMProblems
 
 using AbstractMCMC: AbstractMCMC
+import Random: AbstractRNG
+
```

Comment:
```suggestion
import Distributions: logpdf

```

otherwise `observation_logdensity()` will error